envi/archs/amd64/opcode64.py

```diff
@@ -378,7 +378,7 @@
 ( 0, INS_SYSTEM, ADDRMETH_G | OPTYPE_v | OP_W, ADDRMETH_E | OPTYPE_w | OP_R, ARG_NONE, ARG_NONE, cpu_80386, "lar", 0, 0, 0),  
 ( 0, INS_SYSTEM, ADDRMETH_G | OPTYPE_v | OP_W, ADDRMETH_E | OPTYPE_w | OP_R, ARG_NONE, ARG_NONE, cpu_80386, "lsl", 0, 0, 0),  
 ( 0, 0, 0, 0, 0, 0, 0, 0, 0, 0),  
-( 0, INS_SYSTEM, 0, 0, 0, ARG_NONE, cpu_AMD64, "syscall", 0, 0, 0),
+( 0, INS_SYSCALL, 0, 0, 0, ARG_NONE, cpu_AMD64, "syscall", 0, 0, 0),
```

Comment:
want to be able to identify key instructions (particularly those which we can't emulate very well) by opcode number, not mnem string comparisons.